build/ci/golangci.yml

```diff
@@ -51,6 +51,8 @@ linters-settings:
       - ^fmt\.Errorf$(# Use errors\.Errorf instead to include stack trace)?
       # No "gonanoid" package, use "internal/pkg/idgenerator" package instead
       - ^gonanoid\..*$(# Use "internal/pkg/idgenerator" package instead)?
+      # No logger calls without context
+      #- ^logger\.(Log|Debug|Info|Warn|Error|Debugf|Infof|Warnf|Errorf)$(# Use a method with Ctx suffix instead)?
```

Comment:
Is there some better way to disallow the old methods? This depends on the variable name `logger` and doesn't catch cases such as `root.logger.Infof`. It should not matter how the logger is obtained.

Reply:
https://stackoverflow.com/questions/59627756/is-there-a-way-to-block-the-usage-of-a-specific-function-from-a-package-when-lin

It may be ok to disable all Info/Info calls.
These method names we probably only use within the logger.